proto/lighthouse-result.proto

```diff
@@ -377,6 +377,10 @@ message AuditResult {
   // The unit of the numeric_value field. Used to format the numeric value for
   // display.
   string numeric_unit = 12;
+
+  // [EXPERIMENTAL] Estimates of how much this audit affects various performance
+  // metrics. Values will be in the unit of the respective metrics.
+  google.protobuf.Struct _metricSavings = 13;
```

Comment:
Is renaming this to `metricSavings` going to cause issues down the line?

Reply:
(otherwise it's my understanding that it'll have to ship in the proto forever, just not be populated anymore)